cmd/buildctl/build_test.go

```diff
@@ -111,6 +112,58 @@ func testBuildContainerdExporter(t *testing.T, sb integration.Sandbox) {
 	require.Equal(t, ok, true)
 }
 
+func testBuildMetadataFile(t *testing.T, sb integration.Sandbox) {
+	cdAddress := sb.ContainerdAddress()
+	if cdAddress == "" {
```

Comment:
You don't need to check for image existence in this test. It is covered in others. Or if you want to then move this containerd check to line 155 so that at least rest of this test runs in all workers.